frontend/packages/local-storage-operator-plugin/integration-tests/tests/disks-list-page.scenario.ts

```diff
@@ -0,0 +1,87 @@
+import { resourceRows, isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { clickNavLink } from '@console/internal-integration-tests/views/sidenav.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  page,
+  diskList,
+  clickFilterDropdown,
+  checkDiskFilter,
+  clearDiskFilter,
+} from '../views/disks-list-page.views';
+import { LOCAL_STORAGE_NAMESPACE } from '../../src/constants/index';
+import {
+  LocalVolumeDiscoveryResultKind,
+  DiskMetadata,
+  DiskStates,
+} from '../../src/components/disks-list/types';
+
+// Prerequisite: Local volume discovery to be present
+describe('Disk list is accessible from Nodes view', () => {
+  let discoveredDevices: DiskMetadata[];
+  beforeAll(async () => {
+    const lvdrJson = JSON.parse(
+      execSync(
+        `kubectl get localvolumediscoveryresults -n ${LOCAL_STORAGE_NAMESPACE} -o json`,
+      ).toString(),
+    );
+    const lvdr: LocalVolumeDiscoveryResultKind = lvdrJson.items[0];
+    const { nodeName } = lvdr.spec;
+    discoveredDevices = lvdr.status.discoveredDevices;
+    await browser.get(`${appHost}/`);
+    await clickNavLink(['Compute', 'Nodes']);
+    await isLoaded();
+    click($(`a[data-test-id="${nodeName}"]`));
+  });
+  it('`Disks` tab is enabled and accessible', async () => {
+    await browser.wait(until.presenceOf(page.diskTab));
+    click(page.diskTab);
+  });
+  it('List title is `Disks`', async () => {
+    await isLoaded();
+    expect(page.resourceTitles.get(1).getText()).toBe('Disks');
+  });
+  it('Headers are displayed with correct labels', () => {
+    diskList.headers.forEach(async (header) => {
+      const headers = await $(`th[data-label="${header}"]`);
+      expect(headers.isPresent()).toBeTruthy();
+    });
+  });
+  it('Rows are displayed with correct data format', async () => {
+    const rows = resourceRows;
+    expect(rows.count()).toBe(discoveredDevices.length);
+    const firstRow = rows.first().$$('td');
+    const firstDisk = discoveredDevices[0];
+    expect(firstRow.get(0).getText()).toBe(firstDisk.path);
+    expect(firstRow.get(1).getText()).toBe(firstDisk.status.state);
+    expect(firstRow.get(2).getAttribute('innerText')).toBe(firstDisk.type || '-');
+    expect(firstRow.get(3).getAttribute('innerText')).toBe(firstDisk.model || '-');
+    /* @TODO: (afreen): importing `units.js` is creating a cyclic dependency. Fix the imports and modify test condition */
+    expect(firstRow.get(4).getAttribute('innerText')).not.toBe('-');
+    expect(firstRow.get(5).getAttribute('innerText')).toBe(firstDisk.fstype || '-');
+  });
+  it('Text filter is accessible and working', async () => {
+    const textbox = await diskList.textFilter;
+    await textbox.sendKeys(discoveredDevices[0].path);
+    let updatedRows = resourceRows.count();
+    expect(updatedRows).toBe(1);
+    // info: clear() is not triggering event to clear the filter and update rows
```

Comment:
I don't see clear method being called here. So I guess this comment can be removed.

Reply:
https://github.com/openshift/console/pull/6606#discussion_r487679934

Reply:
See the lines 71-72, its for that and its valid here.